tests/system/providers/amazon/aws/example_s3_to_sql.py

```diff
@@ -44,6 +64,45 @@
 """
 
 
+@task
+def create_connection(conn_id_name: str, cluster_id: str):
+    cluster_endpoint = RedshiftHook().conn.describe_clusters(ClusterIdentifier=cluster_id)["Clusters"][0]
```

Comment:
any particular reason to choose Redshift to host the SQL DB as opposed to AWS RDS for instance ?
just genuine curiosity, I really don't know which one makes more sense/is easier to setup

Reply:
To be entirely honest, I'm open to suggestions.  The existing sql_to_s3 test used Redshift as the source, so I did the same here.

Reply:
I don't think there'd be any particular reason to choose one over the other, as long as we get an SQL database. I think it makes sense to use this because its been used for several other tests in our package.